app/assets/stylesheets/newflow_layout.scss

```diff
@@ -18,82 +44,346 @@ body {
 #signinup-form {
     max-width: 75rem;
     border-top: 0;
-    margin-top: 3rem;
     margin-left: auto;
     margin-right: auto;
+    margin-bottom: 3rem;
+    margin-top: 4rem;
+    padding-left: 14px;
```

Comment:
Should use `rem` instead of `px` (here and elsewhere)?

Reply:
Done.